trieste/acquisition/function/function.py

```diff
@@ -36,7 +36,10 @@
     SingleModelVectorizedAcquisitionBuilder,
 )
 
-M_contra = TypeVar("M_contra", bound=ProbabilisticModel, contravariant=True)
+ProbabilisticModelType = TypeVar(
+    "ProbabilisticModelType", bound=ProbabilisticModel, contravariant=True
+)
+""" Contravariant type variable bound to :class:`~trieste.models.ProbabilisticModel`. """
```

Comment:
Could they all be defined somewhere else once?

Reply:
Good suggestion. Done.